autobahn/wamp/protocol.py

```diff
@@ -58,6 +59,17 @@ def is_method_or_function(f):
     return inspect.ismethod(f) or inspect.isfunction(f)
 
 
+def check_args_types(func, *args, **kwargs):
+    # Converge both args and kwargs into a dictionary
+    arguments = inspect.getcallargs(func, *args, **kwargs)
+    response = []
+    for name, kind in typing.get_type_hints(func).items():
```

Comment:
We could do without importing `typing` there as well by using `func.__annotations__` instead.